index.css

```diff
@@ -42,7 +42,7 @@ input::-webkit-outer-spin-button {
     display:flex;
     flex-direction:row;
      border: 2px soild white;
-     color:white;
+     color:red;  /*  here is the small change*/
      font-weight: bold;
   }
   #sum1,#sum2,#idly1,#idly2,#puri1,#puri2,#bun1,#bun2,#total{
```

Comment:
If you are putting same styling for all the elements, then make it a single class

Reply:
If you are putting same styling for all the elements, then make it a single class